### PR TITLE
Proofreading May 2021

### PR DESCRIPTION
**Pull request by ThrawnCA**
Results of a recent proofreading pass.

- Fix missing/mismatched quotes.
- Ensure commas before dialogue.
- Ensure incantations are in italics.
- Ensure punctuation after an italicised word is also italic.
- Fix capitalisation of The Burrow.
- Minor Britpicking.
- Fix numbering of chapter 9.
- Fix reviving spell name (JKR initially erred and later corrected it).
- Fix small continuity error regarding the need to actually revise when Harry and Hermione are going over his future memories (they revise beforehand, then have a discussion afterward about whether there's any need for him to revise).
- Adjust timing during the duelling tournament; surely spectators were not watching Harry try to stand for multiple minutes.
- Miscellaneous spelling/grammar/punctuation fixes.

Something I wasn't sure about was the fact that Sirius' collection of Horcruxes makes no mention of the diadem, yet Harry then states that they have everything except the snake. Possibly it could just be added to the list in chapter 31?

### CODE REVIEW

File: src/OEBPS/Chapter17.xhtml [hunk 1]
```diff
@@ -164,7 +164,7 @@
...
-            <p>Her brothers all left under that glare, but Harry wouldn’t. After the way he reacted to Dad after they returned from Diagon Alley, Ginny did not expect Harry to stand up to her parents the way he did. He was never disrespectful, but he was… very firm. He explained why Hogwarts wasn’t always safe, though her parents required more details than she had. Oddly enough, while Mum got angry, Dad stayed very calm. He looked at Harry thoughtfully and asked if he’d thought it all out ahead of time.</p>
+            <p>Her brothers all left under that glare, but Harry wouldn’t. After the way he reacted to Dad after they returned from Diagon Alley, Ginny did not expect Harry to stand up to her parents the way he did. He was never insulting, but he was… very firm. He explained why Hogwarts wasn’t always safe, though her parents required more details than she had. Oddly enough, while Mum got angry, Dad stayed very calm. He looked at Harry thoughtfully and asked if he’d thought it all out ahead of time.</p>
```
Comment (IntermittentlyRupert) [inline src/OEBPS/Chapter17.xhtml]: I don't think the original is wrong here, so I'd prefer to leave the original.

**Comment by IntermittentlyRupert** [comment]
Re the horcrux list in ch31, I'm in favour of leaving it as is until/unless it comes up again. It could be an oversight on Matthew's part, but it could also be omitted because the diadem is at Hogwarts at that point in the timeline.

ETA: And/or the Sorting Hat may just be wholesale replacing the diadem for the purposes of this fic? The details there are a bit fuzzy, I think, because this was started before Deathly Hallows came out.

ETA2: Also Nagini probably isn't actually a Horcrux yet by that point in the timeline, I don't think. 😄

File: src/OEBPS/Chapter17.xhtml (hunk 1 above)
Comment (ThrawnCA) [reply src/OEBPS/Chapter17.xhtml]: Well, he did cut them off and speak over them a bunch of times, which is generally considered disrespectful. That's why I changed it.

        “Harry, I understand things have been difficult for you, but you mustn’t let—”
        “I’m sorry you feel that I have been taken advantage of,” Harry interjected coldly. “It’s sad that you have so little faith in your daughter. After our discussion on Saturday, I was hoping we wouldn’t have to have another argument about the second item you and Mr. Weasley agreed to.”

        Mrs. Weasley shook her head, “Harry, it isn’t—”
        Harry cut her off. “It isn’t that dangerous?

        “But he’s gone now, you said—”
        “He’s gone – for now,” Harry agreed, cutting her off.

But I suppose it's written from Ginny's perspective, and she might see those differently.

**Comment by ThrawnCA** [comment]
> Re the horcrux list in ch31, I'm in favour of leaving it as is until/unless it comes up again. It could be an oversight on Matthew's part, but it could also be omitted because the diadem is at Hogwarts at that point in the timeline.

Well, Harry specifically states in chapter 31 that "that left only Nagini and the Sorting Hat". So no, the Diadem isn't sitting at Hogwarts. Of course, it wasn't confirmed as a horcrux in book 6.

> ETA: And/or the Sorting Hat may just be wholesale replacing the diadem for the purposes of this fic? The details there are a bit fuzzy, I think, because this was started before Deathly Hallows came out.

The author notes are gone now, but I think I recall the Author Lord stating that the Sorting Hat was an extra one, and it made him more unstable, triggering his assault on Hogwarts.

> ETA2: Also Nagini probably isn't actually a Horcrux yet by that point in the timeline, I don't think. 😄

Eh, Harry probably isn't sure, but would want to do away with the snake anyway.

**Pull request merged**